### PR TITLE
ARM64: Fix Build Test Break

### PR DESCRIPTION
**Pull request by kyulee1**

### CODE REVIEW

**Comment by kyulee1** [comment]
@dotnet-bot test Windows_NT arm64 checked

**Comment by kyulee1** [comment]
@dotnet-bot test Windows_NT arm64 release

**Comment by kyulee1** [comment]
@Priya91 @dotnet/arm64-contrib PTAL

File: build-test.cmd [hunk 1]
```diff
@@ -191,7 +190,7 @@ if defined __UpdateInvalidPackagesArg (
   set __up=-updateinvalidpackageversions
 )
 
-call %~dp0run.cmd build -buildManagedTests -MsBuildLog=!__msbuildLog! -MsBuildWrn=!__msbuildWrn! -MsBuildErr=!__msbuildErr! %__up% %__RunArgs% %__unprocessedBuildArgs%
+call %__ProjectDir%\run.cmd build -buildManagedTests -MsBuildLog=!__msbuildLog! -MsBuildWrn=!__msbuildWrn! -MsBuildErr=!__msbuildErr! %__up% %__RunArgs% %__unprocessedBuildArgs%
 if errorlevel 1 (
```
Comment (Priya91) [inline build-test.cmd]: The -buildmanagedTests uses relative paths in config.json. Replace this with ```-Project=%__ProjectDir%/tests/build.proj``` and remove the -buildManagedTests flag from ```config.json```

File: build-test.cmd [hunk 2]
```diff
@@ -212,7 +211,7 @@ set __msbuildLog=/flp:Verbosity=normal;LogFile="%__BuildLog%"
 set __msbuildWrn=/flp1:WarningsOnly;LogFile="%__BuildWrn%"
 set __msbuildErr=/flp2:ErrorsOnly;LogFile="%__BuildErr%"
 
-call %~dp0run.cmd build -testOverlay -MsBuildLog=!__msbuildLog! -MsBuildWrn=!__msbuildWrn! -MsBuildErr=!__msbuildErr! %__RunArgs% %__unprocessedBuildArgs%
+call %__ProjectDir%\run.cmd build -testOverlay -MsBuildLog=!__msbuildLog! -MsBuildWrn=!__msbuildWrn! -MsBuildErr=!__msbuildErr! %__RunArgs% %__unprocessedBuildArgs%
 if errorlevel 1 (
```
Comment (Priya91) [inline build-test.cmd]: This should be ```call %__ProjectDir%\run.cmd build -Project=%__ProjectDir%/tests/runtest.proj -testOverlay -MsBuildLog=!__msbuildLog! -MsBuildWrn=!__msbuildWrn! -MsBuildErr=!__msbuildErr! %__RunArgs% %__unprocessedBuildArgs%``` and then in config.json remove the Project setting from testOverlay alias, [here](https://github.com/dotnet/coreclr/blob/master/config.json#L336)

**Comment by Priya91** [comment]
LGTM, with comments.

**Comment by kyulee1** [comment]
@dotnet-bot test Windows_NT arm64 release

**Comment by kyulee1** [comment]
@dotnet-bot test Ubuntu x64 Checked Build and Test
@dotnet-bot test OSX x64 Checked Build and Test

**Comment by kyulee1** [comment]
Fixes are updated. PTAL.

**Comment by Priya91** [comment]
LGTM

**Pull request merged**